README.md

```diff
@@ -23,7 +23,7 @@ Then, install the OpenZeppelin SDK running:
 npm install --global @openzeppelin/cli
 ```
 
-> If you get an `EACCESS permission denied` error while installing, please refer to the [npm documentation on global installs permission errors](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally). Alternatively, you may run `sudo npm install --unsafe-perm --global @openzeppelin/cli`, but this is highly discouraged, and you should rather either use a node version manager or manually change npm's default directory.
+> If you get an `EACCESS permission denied` error while installing, please refer to the [npm documentation on global installs permission errors](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally). 
```

Comment:
Let's not remove this. We are already mentioning it is highly discouraged, and it may help getting a user to run this when in a rush (such as in a hackathon!).